red_october/fields/red_october_char.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+import psycopg2
+
+from odoo import fields
+
+from odoo.tools import human_size
+
+
+_logger = logging.getLogger(__name__)
+
+
+class RedOctoberChar(fields.Field):
+
+    type = 'red_october'
+    column_type = None
+
+    def convert_to_column(self, value, record):
+        return
+
+    def convert_to_cache(self, value, record, validate=True):
+        if isinstance(value, buffer):
+            return str(value)
+        return value
+
+    def read(self, records):
+        # values are stored in attachments, retrieve them
+        attachments = self._get_attachments(records).with_context(
+            bin_size=False,
+        )
+        data = {att.res_id: att.datas for att in attachments}
+        _logger.debug(data)
+        for record in records:
+            record._cache[self.name] = data.get(record.id, False)
```

Comment:
Mainly just that `False` seems to correspond with what Odoo typically puts as a blank field value. Bugs the hell out of me personally, so let me know if this isn't the case.